Solutions/Cribl/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{  
+    "publisherId": "criblinc1673975616879",
+	"offerId": "cribl_stream_vm",
```

Comment:
Include sentinel keyword in offer id 

![image](https://github.com/user-attachments/assets/00dff7be-1ca2-4e33-8060-e80a7176a3f0)

Reply:
resolved, added a new application that follows the integration requirements, 'microsoft-sentinel-solution-cribl.'